.gitignore

```diff
@@ -42,6 +42,7 @@
 /examples/pmem_and_default_kind
 /examples/pmem_config
 /examples/pmem_cpp_allocator
+/examples/fixed_cpp_allocator
```

Comment:
Please, meld this commit with previous commits. Adding separate commits for review fixes can easily lead to a large amount of commits containing fixes. It's unnecessary burden given it's still a PR and commits can be amended anytime you want until the merge with the master branch. Please, sweep through the history of commits - there are no such commits as this one.